src/angular/styles/_typography.scss

```diff
@@ -1068,12 +1083,12 @@ textarea {
   }
 
   &:is(:focus, .sbb-focused) {
-    border-color: var(--sbb-color-weak-accent);
+    border-color: var(--sbb-form-border-color-active);
```

Comment:
not sure if we should remove --sbb-color-weak-accent here? related to other comments of me :D

Reply:
It is not the same color as the week accent we chose for the weak accent in darkmode so its fine that way :D

Reply:
@jeripeierSBB is correct:
`--sbb-color-weak-accent` has `--sbb-color-iron` in Lean mode and `--sbb-color-granite` in standard mode.
You also changed the focus color of the form field in standard mode what is not intended.

I suggest to set `--sbb-form-border-color-active` to `var(--sbb-color-weak-accent)` in line 234 and to override `--sbb-form-border-color-active` for the dark mode.